tests/flow/base.py

```diff
@@ -46,8 +46,20 @@ def _assert_actual_results_contained_in_expected_results(self,
 
 
     def _assert_resultset_equals_expected(self, actual_result, query_info):
-        self._assert_only_expected_results_are_in_actual_results(actual_result, query_info)
-        self._assert_actual_results_contained_in_expected_results(actual_result, query_info, len(query_info.expected_result))
+        actual_result_set = []
+        if actual_result.result_set is not None:
+            actual_result_set = actual_result.result_set
+
+        # Assert number of results.
+        self.env.assertEqual(len(actual_result_set), len(query_info.expected_result))
+
+        # Assert actual values vs expected values.
+        for res in query_info.expected_result:
+            self.env.assertIn(res, actual_result_set)
```

Comment:
since it is a result set, the order does not matter, and mutual inclusion should be sufficient. unless we have ORDER BY
I'll add another function for ordered comparison